elements_imaging/imaging.py

```diff
@@ -143,7 +141,8 @@ class ProcessingTask(dj.Manual):
     -> scan.Scan
     -> ProcessingParamSet
     ---
-    task_mode='load': enum('load', 'trigger')  # 'load': load computed analysis results, 'trigger': trigger computation
+    processing_output_dir: varchar(255)         #  output directory of the processed scan relative to root data directory
```

Comment:
How should we modify `workflow-imaging/workflow_imaging/ingest.py` to find the `processing_output_dir`?

Reply:
Should we add another `csv` file to `workflow-imaging/user_data` for the `processing_output_dir` and `curation_output_dir`?

Reply:
`processing_output_dir` will be manually provided to us by the user, as this `ProcessingTask` table is now a manual table